examples/plotting/masked_composite_plot.py

```diff
@@ -0,0 +1,57 @@
+"""
+================================
+Creating a Masked Composite Plot
+================================
+
+This example combines creating a composite plot with masking out the
+solar disk. The purpose is to introduce users to the plotting process
+required to overlay multiple Maps that have been modified with other
+**sunpy** functions. The resulting plot in this tutorial shows information
+on the upper photosphere, quiet corona, and magnetogram contours.
+"""
+import matplotlib.pyplot as plt
+
+import astropy.units as u
+
+import sunpy.data.sample
+from sunpy.map import Map
+from sunpy.map.maputils import all_coordinates_from_map, coordinate_is_on_solar_disk
+
+###############################################################################
+# Let's import sample data representing the three types of data we want to
+# overlay: AIA 1600 (upper photosphere), AIA 171 (quiet corona), and HMI
+# data (magnetic fields in the photosphere).
+aia_1600 = Map(sunpy.data.sample.AIA_1600_IMAGE)
+aia_171 = Map(sunpy.data.sample.AIA_171_IMAGE)
+hmi = Map(sunpy.data.sample.HMI_LOS_IMAGE)
+
+###############################################################################
+# Next, let's mask out the solar disk of the AIA 171 image since we will be
+# overlaying the AIA 1600 photosphere.
+hpc_coords = all_coordinates_from_map(aia_171)
+mask = coordinate_is_on_solar_disk(hpc_coords)
+palette = aia_171.cmap.copy()
```

Comment:
```suggestion
colormap = aia_171.cmap.copy()
```
I've never heard "palette" before, so I think best to stick to "colormap" which is a more common term here?